Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -1660,105 +1665,185 @@ private void UpdateTrainBaseResistance()
-                            StartFrictionHigh = 15.93f;  // Starting friction for a 100 ton(US) car with standard roller bearings, not snowing
+                            // Set to snowing (frozen value)
+                            StartFrictionLowLoad = 12.771f;  // Starting friction for a 10 ton(US) car with standard roller bearings, snowing
+                            StartFrictionHighLoad = 30.0f;  // Starting friction for a 100 ton(US) car with standard roller bearings, snowing
+                        }
+                        else if (WheelBearingTemperatureDegC > 25)
+                        {
+                            // Set to normal temperature value
+                            StartFrictionLowLoad = 4.257f;  // Starting friction for a 10 ton(US) car with standard roller bearings, not snowing
+                            StartFrictionHighLoad = 15.93f;  // Starting friction for a 100 ton(US) car with standard roller bearings, not snowing
                         }
                         else
                         {
-                            StartFrictionLow = 12.771f;  // Starting friction for a 10 ton(US) car with standard roller bearings, snowing
-                            StartFrictionHigh = 30.0f;  // Starting friction for a 100 ton(US) car with standard roller bearings, snowing
+                            // Set to variable value as bearing heats and cools
+                            StartFrictionLowLoad = LowGrad * WheelBearingTemperatureDegC + LowIntersect;
+                            StartFrictionHighLoad = HighGrad * WheelBearingTemperatureDegC + HighIntersect;
                         }
+
                         if (Kg.ToTUS(MassKG) < 10.0)
                         {
-                            StaticFrictionFactorLb = StartFrictionLow;  // Starting friction for a < 10 ton(US) car with standard roller bearings
+                            StaticFrictionFactorLb = StartFrictionLowLoad;  // Starting friction for a < 10 ton(US) car with standard roller bearings
                         }
                         else if (Kg.ToTUS(MassKG) > 100.0)
                         {
-                            StaticFrictionFactorLb = StartFrictionHigh;  // Starting friction for a > 100 ton(US) car with standard roller bearings
+                            StaticFrictionFactorLb = StartFrictionHighLoad;  // Starting friction for a > 100 ton(US) car with standard roller bearings
                         }
                         else
                         {
-                            StaticFrictionFactorLb = (((Kg.ToTUS(MassKG) - 10.0f) / 90.0f) * (StartFrictionHigh - StartFrictionLow)) + StartFrictionLow;
+                            StaticFrictionFactorLb = (((Kg.ToTUS(MassKG) - 10.0f) / 90.0f) * (StartFrictionHighLoad - StartFrictionLowLoad)) + StartFrictionLowLoad;
                         }
```

Comment:
This should be brought to metric units here, which is default for OR, if there aren't strong reasons against.

Reply:
Ok, converted.